dotnet/test/ColumnStoreDotNetTest.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.0" />
+    <PackageReference Include="xunit" Version="2.4.0" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.0" />
+    <ProjectReference Include="..\src\ColumnStoreDotNet.csproj" />
+  </ItemGroup>
+  
+  <Target Name="TestTarget" AfterTargets="Build">
+	  <Copy SourceFiles="..\lib-linux\dotnet_mcsapi.so" DestinationFolder="$(OutputPath)" ContinueOnError="false" Condition=" '$(OS)' == 'Unix' " />

```

Comment:
change to "..\lib-linux-x64\dotnet_mcsapi.so" to be consistent with the documentation in README.md